mm2src/coins/for_tests/RICK_2c33baf0c40eebcb70fc22eab0158e315e2176e4a3f20acddcd849186fca492c.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I think it's better to have one json file for HD wallet test transactions similar to `tBCH_tx_history_fixtures.json`

Reply:
Done